jobserver/api/releases.py

```diff
@@ -67,19 +68,22 @@ def maybe_create_report(rfile, filename, user):
     """
     name, ext = os.path.splitext(filename)
     analysis_request = AnalysisRequest.objects.filter(pk=name).first()
-    if ext == ".html" and analysis_request:
-        report = Report.objects.create(
-            project=rfile.workspace.project,
-            release_file=rfile,
-            title=analysis_request.title,
-            description="TODO fill from AR",
-            created_by=user,
-            updated_by=user,
-        )
-        analysis_request.report = report
-        analysis_request.save(update_fields=["report"])
-        # TODO: notify the AR creator a report has been created
-        # notify some staff…?
+    if not (ext == ".html" and analysis_request):
+        return
+
+    report = Report.objects.create(
+        project=rfile.workspace.project,
+        release_file=rfile,
+        title=analysis_request.title,
+        description="TODO fill from AR",
+        created_by=user,
+        updated_by=user,
+    )
+    analysis_request.report = report
+    analysis_request.save(update_fields=["report"])
+
+    send_report_uploaded_notification(analysis_request)
+    # TODO: notify some staff…?
```

Comment:
Super tiny nitpick: I'm not sure if we'll need to notify anyone else, so this TODO is probably redundant.